packages/client/src/index.ts

```diff
@@ -151,7 +181,8 @@ export default class LogtoClient {
    */
   async fetchUserInfo(): Promise<UserInfoResponse> {
     const { userinfoEndpoint } = await this.getOidcConfig();
-    const accessToken = await this.getAccessToken();
+    // eslint-disable-next-line unicorn/no-useless-undefined
+    const accessToken = await this.getAccessToken(undefined, undefined);
```

Comment:
What is the purpose of these two undefined?

Reply:
was trying to use `undefined` for more clear function calling, but removed now.